launch/cartographer.launch

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" ?>    
+<!--
+  Copyright 2016 The Cartographer Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<launch>
+    <arg name="robotont_ns" default="robotont"/>
+    <node name="cartographer_node" pkg="cartographer_ros"
+        type="cartographer_node" args="
+        -configuration_directory $(find robotont_demos)/config
+        -configuration_basename robotont_cartographer_localization_2d.lua"
+        output="screen">
+        <remap from="odom" to="$(arg robotont_ns)/odom"/>
```

Comment:
I guess this falls into the big namespacing discussion, but can't we avoid the robotont_ns argument here by putting the whole node under ns=robotont namespace? In general, namespacing in ROS is designed not to be managed through custom variables in topic names.